### ISSUE TITLE
More icons for FFXIV Macros

### ISSUE THREAD

**Issue by GunB** [goatcorp/suggestions#605]
### Description
More icons for the built-in macros system

### More info
Do you guys know if would be possible to create a plugin to add more Icons for the macros? With so many plugins I'm running out of them... Maybe a fontawesome of icons to pick for different actions. 

The game has inheritably a way to /macroicon of minions, skills and items. Maybe is possible to add another category to the list. I'm a dirty controller player ok? ;-; we have feelings too

Or maybe a way to make the icon not show deactivated in the hotbar/crossbar when using a minion or an item or an unavailable skill as we are using it just for the icon and not for the function

![image](https://user-images.githubusercontent.com/3722960/136066970-09ef7402-165e-4a42-a514-54c6c424c042.png)

----

 - [x] I checked if this request wasn't already proposed
 - [x] I made sure this request follows Dalamud's [plugin guidelines](https://github.com/goatcorp/FFXIVQuickLauncher)

**Comment by someONEgood** [comment]
This isn't exactly a direct solution to your problem, but have you tried using the QoL bars plugin?

**It allows for macro creation right inside the bar:**
![qolmacro](https://user-images.githubusercontent.com/46905309/137595750-bd539ebf-bca0-479f-b3b7-f5cfa7c46ad4.JPG)
**The magnifying glass in the corner is the icon browser ____________^^^**

![qol](https://user-images.githubusercontent.com/46905309/137595505-84ad7a0b-bd8b-4012-a9ec-b81b22b281d0.JPG)

 **and you can select from any icon within the game from a really well designed selection menu:**
![060756](https://user-images.githubusercontent.com/46905309/137594957-887d2fd4-68f3-41f7-9070-2b4f739c10ec.JPG)

 This won't allow for said macros/abilities to be put on your cross-bar (I also use controller, it's fantastic with proper setup so don't deride yourself), but if you want to be at your best, you shouldn't use macros for abilities anyways due to the fact that they don't que. I hope this was helpful, even if it was 11 days later.

**Comment by GunB** [comment]
Yes, I know QoL plugin exists and sadly this doesn't solve my ticket in any way. I also don't want to use macros for abilities but to have access to the quick toggles I like to use in-game. Like the sound setter or invite or many others I have. 
I don't like using my keyboard and mouse for this game. I only use it for very specific configurations and then let them all set in macros to use them with my gamepad.
